easybuild/easyconfigs/s/Score-P/Score-P-6.0-gompi-2020a.eb

```
@@ -0,0 +1,60 @@
+##
+# This is an easyconfig file for EasyBuild, see https://github.com/easybuilders/easybuild
+# Copyright:: Copyright 2013-2019 Juelich Supercomputing Centre, Germany
+# Authors::   Bernd Mohr <b.mohr@fz-juelich.de>
+#             Markus Geimer <m.geimer@fz-juelich.de>
+# License::   3-clause BSD
+#
+# This work is based on experiences from the UNITE project
+# http://apps.fz-juelich.de/unite/
+##
+
+name = 'Score-P'
+version = '6.0'
+
+homepage = 'https://www.score-p.org'
+description = """
+ The Score-P measurement infrastructure is a highly scalable and easy-to-use
+ tool suite for profiling, event tracing, and online analysis of HPC
+ applications.
+"""
+
+toolchain = {'name': 'gompi', 'version': '2020a'}
+
+source_urls = ['https://www.vi-hps.org/cms/upload/packages/scorep/']
+sources = ['scorep-%(version)s.tar.gz']
+checksums = [
+    '5dc1023eb766ba5407f0b5e0845ec786e0021f1da757da737db1fb71fc4236b8',  # scorep-6.0.tar.gz
+]
+
+builddependencies = [
+    ('CubeLib', '4.4.4'),
+    ('CubeWriter', '4.4.3'),
+    # Unwinding/sampling support (optional):
+    ('libunwind', '1.3.1'),
+]
+
+dependencies = [
+    # binutils is implicitly available via GCC toolchain
+    # ('binutils', '2.34'),
```

These build deps should be moved to runtime dependencies as discussed in #10397 and #10496

```suggestion
dependencies = [
    ('CubeLib', '4.4.4'),
    ('CubeWriter', '4.4.3'),
    # Unwinding/sampling support (optional):
    ('libunwind', '1.3.1'),
```

---

fixed in 137f877

---

and also for OTF2's SIONlib dep in 91a3bff